File: Conditional Statement/Problem 4. Multiplication Sign/MultiplicationSign.cs
using System;
//Problem 4. Multiplication Sign
//Write a program that shows the sign (+, - or 0) of the product of three real numbers, without calculating it.
//Use a sequence of if operators.
class MultiplicationSign
{
    static void Main()
    {
        Console.WriteLine("The program shows the sign of the product of three numbers");
        int a = int.Parse(Console.ReadLine());
        int b = int.Parse(Console.ReadLine());
        int c = int.Parse(Console.ReadLine());

        if (a * b * c < 0)
        {
            Console.WriteLine("-");
        }
        else if (a * b * c > 0)
        {
                Console.WriteLine("+");
        }
        else if (a * b * c == 0)
        {
            Console.WriteLine("0");
        }
    }
}

File: Javascript Fundamentals/Conditional statements/js/MultiplicationSign.js
//Write a script that shows the sign (+, - or 0) of the product of three real numbers, without calculating it.
//Use a sequence of if operators.
//var a = console.prompt("Enter variable");
//var b = console.prompt("Enter variable");
//var c = console.prompt("Enter variable");


var a = -3;
var b = -2;
var c = -1;
var result = "Check your entered a, b and c!";

var positive = true;
if (a >= 0) {
	if (b > 0 && c < 0) {
		positive = false;
	}
	else if (b < 0 && c > 0) {
		positive = false;
	}
}
else {
	if (b > 0 && c > 0) {
		positive = false;
	}
	else if (b < 0 && c < 0) {
		positive = false;
	}
}
if (positive) {
	result = "The product will be positive!";
}
else {
	result = "The product will be negative!";
}
console.log(result);
